### DISCUSSION TITLE
Broken links to docs in index.md (possibly referenced in #2150)

### DISCUSSION THREAD

**Discussion by geographyclub** [maplibre/maplibre-gl-js#4227] [General]
Hi all, I'm a new contributor and making my way thru the docs.

I see that the links on [index.md](https://github.com/maplibre/maplibre-gl-js/blob/main/docs/index.md#reading-this-documentation] are broken, like [main](https://github.com/maplibre/maplibre-gl-js/blob/main/docs/API/README.md)

This issue might have been mentioned in #2150.

What does it take to start fixing these links?

**Comment by HarelM** [comment]
The docs folder gets populated with markdown files as part of the docs building process.
The output can be found here:
https://maplibre.org/maplibre-gl-js/docs/

index.md is just the intro to these docs, you can read the readme.md in that folder for more information. 

After learning this, if you still think there's a problem, feel free to open a PR with the relevant fix, I'll be happy to review it!

**Comment by geographyclub** [comment]
I can change the urls at index.md to point to the generated links.

This being my first pull request, can you offer any advice (besides reading contributing.md)?

Thanks for taking the time.

**Comment by HarelM** [comment]
There are readme files scattered around the repo, each with their own folder and explanations, I suggest reading those.
As part of the documentation generation the process checks for link validity, so I'm not sure there's a real issue here, but feel free to poke around. Thanks!
